dev/com.ibm.ws.microprofile.config_fat/test-applications/simultaneousRequests.war/src/com/ibm/ws/microprofile/appConfig/simultaneousRequests/test/ConfigHolder.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.appConfig.simultaneousRequests.test;
+
+import javax.inject.Inject;
+import javax.enterprise.context.RequestScoped;
+
+import org.eclipse.microprofile.config.Config;
+
+
+@RequestScoped
+public class ConfigHolder {
```

Comment:
What's this class for?

Reply:
Ah, that's a leftover from trying to get the test to work. I kept it because something needs a bean defining annotation to enable CDI, I'll rename and comment